src/main/scala/codacy/checkstyle/Checkstyle.scala

```diff
@@ -42,6 +44,20 @@ object Checkstyle extends Tool {
     run(filesToLint, config)
   }
 
+  /**
+    * Checkstyle plugin defines a config_loc property that can be used in Checkstyle configuration
+    * files to define paths to other config files like suppressions.xml.
+    * We should define it for users to be able to use this property on the configuration file.
+    *
+    * @see <a href="https://docs.gradle.org/6.5.1/userguide/checkstyle_plugin.html#sec:checkstyle_built_in_variables">Checkstyle built in vars</a>
+    *
+    * @param configFilePath the path to checkstyle configuration file§
+    */
+  private def setCheckstyleConfigLocationProperty(configFilePath: String): Unit = {
```

Comment:
Codacy found an issue: [Use active names for operations with side effects](https://app.codacy.com/gh/codacy/codacy-checkstyle/pullRequest?prid=5929160)